client/naturalcrit/successPage/successPage.jsx

```diff
@@ -31,7 +33,9 @@ const SuccessPage = React.createClass({
 		};
 	},
 	componentDidMount: function() {
-	  setTimeout(function(){window.location='/login';}, 1500);
+		const redirectURL = window.localStorage.getItem(RedirectLocation) || '/';
+		window.localStorage.removeItem(RedirectLocation);
+		setTimeout(function(){window.location=redirectURL;}, 1500);
```

Comment:
local to session storage

Reply:
Updated to sessionStorage in latest commit